### ISSUE TITLE
New DCP profiles - where to keep logs?

### ISSUE THREAD

**Issue by Beep6581** [Beep6581/RawTherapee#3183]
While generating DCP profiles, I also save the log files which contain statistics on the DCPs. The tools available in dcamprof and argyllCMS can be used to regenerate these statistics, and more, so the question is do we need to store these logs? Are they of any use after the commit? @iliasg ?
Commit https://github.com/Beep6581/RawTherapee/commit/583a737
Logs http://filebin.net/bqgvihx5bl

**Comment by iliasg** [comment]
I am not sure if we need them. Please make some of them available for 1-2 days .. it will be interesting to compare the log data vs "total_throughoutput" measures. May be it is good to crosscheck both logs and total_throughoutput" measures to detect if a bad color result comes from the profile or from a possible bug in RT's color management. 
Tomorrow I have time to do some job :)

**Comment by sguyader** [comment]
Hi, I was able to find  some DCP profiles created by Mario (aka "cack")on the Fuji X Forum. He used DCamProf to generate the profiles for the X-E1, X-E2, X-T1 and X-T10 cameras, and made 2 flavours for each camera: one with some contrast, one with a flat response (I asked him how he generated those 2 flavours, I'm waiting for his answer). I also asked him for his permission to distribute these profiles in RT packages, he's fine with that. Here's the link to download them: http://filebin.net/dsov8zfp31 (I just renamed the profiles).
I tried the X-T1 profiles on my raw files and they seem to work correctly.
If someone can take a look, maybe @iliasg, and if they are good it would be nice to include them in RT.
